ui/app/AppLayouts/Profile/stores/ContactsStore.qml

```diff
@@ -16,6 +18,15 @@ QtObject {
         Component.onCompleted: {
             mainModuleInst.resolvedENS.connect(root.resolvedENS)
         }
+
+        readonly property var conn: Connections {
+            target: d.contactsModuleInst
+            function onTrustStatusRemoved(pubKey: string) {
+                const contactDetails = Utils.getContactDetailsAsJson(pubKey, false, false, true)
+                const displayName = SQUtils.Emoji.parse(ProfileUtils.displayName(contactDetails.localNickname, contactDetails.name, contactDetails.displayName, contactDetails.alias))
+                Global.displaySuccessToastMessage(qsTr("Untrusted mark removed for %1").arg(displayName))
```

Comment:
This could be the trusted mark instead actually since `removeTrustStatus` works both ways. As said above, we should pass the new trust status and then we can change the text depending

Reply:
Yeah see above; but in this case we'd need to know the old status. Or, we can change the notification text to something "neutral", e.g. "Trust mark removed..."

Reply:
Done